### PR TITLE
Add new properties to figure shortcode

### PR DESCRIPTION
**Pull request by heitortsergent**
Add new properties to figure shortcode that can help us control how we display images (size and alignment).

### CODE REVIEW

File: docs/sources/write/shortcodes/index.md [hunk 1]
```diff
@@ -280,6 +288,12 @@ In this example, the image's display size is changed to have a maximum width of
 {{</* figure max-width="50%" src="/static/img/docs/grafana-cloud/k8sPods.png" caption="Pod view in Grafana Kubernetes Monitoring" */>}}
 ```
 
+In this example, the image's display size is changed to have a width of 500px, and the `class` and `link-class` properties are used to center the image on the page. The `width` value must have a unit of measurement, such as pixels or percentages.
+
+```markdown
+{{</* figure width="500px" class="w-100p" link-class="w-fit mx-auto d-flex flex-direction-column" src="/static/img/docs/grafana-cloud/k8sPods.png" caption="Pod view in Grafana Kubernetes Monitoring" caption-align="center" */>}}
+```
+
```
Comment (nate-walters-grafana) [inline docs/sources/write/shortcodes/index.md:296]:
```suggestion
In this example, the image's display size is changed to have a maximum width of 500px, and the `class` and `link-class` properties are used to center the image on the page. The original `width` and `height` values from the image are included without any unit of measurement (such as pixels or percentages).

```markdown
{{</* figure src="/static/img/docs/grafana-cloud/k8sPods.png" width="1275" height="738" max-width="500px" class="w-100p" link-class="w-fit mx-auto d-flex flex-direction-column" caption="Pod view in Grafana Kubernetes Monitoring" caption-align="center" */>}}
```

```

**Review by nate-walters-grafana — COMMENTED** [review]
One small fix to the example at the end

File: docs/sources/write/shortcodes/index.md [hunk 2]
```diff
@@ -253,18 +253,26 @@ The `figure` shortcode renders an image with a caption using an HTML [`<figure>`
...
+| `show-caption`  | If set to `"true"`, the rendered `<figure>` includes a `<figcaption>` element with the caption set in _caption_. Defaults to `"true"`.                                                                                                                                                          | no       |
+| `src`           | Sets the source of the image.                                                                                                                                                                                                                                                                   | yes      |
+
+{{% admonition type="note" %}}
+Including the original image dimensions as the 'width' and 'height' properties is highly recommended, as it improves page performance and SEO. These values are _only_ used for determining the image aspect ratio and don't equate to the final displayed size.
```
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:274]: TIL!

**Review by jdbaldry — APPROVED** [review]
Nice, thank you for this :)

File: docs/sources/write/shortcodes/index.md (hunk 1 above)
Comment (heitortsergent) [reply docs/sources/write/shortcodes/index.md]: Thanks @nate-walters-grafana!! I totally missed updating the example after our chat, thanks for looking at it again. 🙏

**Pull request merged**